Error while creating Load Balancer Listener

Using below var.tf.vars ..
```

ibmcloud_api_key = "myapi key
ibmcloud_region = "osa"
ibmcloud_zone = "osa21"
service_instance_id = "0b0af8a7-cbee-4a04-8aba-e92018d763d9"
rhel_image_name =  "rhel-85-12132021"
rhcos_import_image              = true
rhcos_import_image_filename     = "rhcos-413.92.202306140611-0-powervs.ppc64le.ova.gz"
rhcos_import_image_storage_type = "tier1"
system_type =  "s922"
network_name =  "offline-cluster-network"
openshift_install_tarball =  "https://mirror.openshift.com/pub/openshift-v4/multi/clients/ocp/latest-4.13/ppc64le/openshift-install-linux.tar.gz"
openshift_client_tarball =  "https://mirror.openshift.com/pub/openshift-v4/multi/clients/ocp/latest-4.13/ppc64le/openshift-client-linux.tar.gz"
cluster_id_prefix = "ghatwala"
cluster_id        = "413-fips"
cluster_domain = "synergyonpower.com"
setup_squid_proxy = false
setup_snat = true
fips_compliant = true

use_ibm_cloud_services    = true
ibm_cloud_vpc_name        = "offline-clusters-vpc"
ibm_cloud_vpc_subnet_name = "sn-20230407-01"

iaas_classic_username     = "apikey"
iaas_classic_api_key      = "" # if empty, will default to ibmcloud_api_key.
iaas_vpc_region           = "jp-osa" # if empty, will default to ibmcloud_region

## Small Configuration Template

bastion   = { memory = "16", processors = "0.5", "count" = 1 }
bootstrap = { memory = "32", processors = "0.5", "count" = 1 }
master    = { memory = "32", processors = "0.5", "count" = 3 }
worker    = { memory = "32", processors = "0.5", "count" = 2 }

#enable_local_registry      = true  #Set to true to enable usage of local registry for restricted network install
#local_registry_image       = "docker.io/ibmcom/registry-ppc64le:2.6.2.5"
#ocp_release_tag            = "4.11.8-ppc64le"
#ocp_release_name           = "ocp-release"
#release_image_override     = ""

storage_type = "nfs"
volume_size = "300"
rhel_subscription_username = "amitkumar_ghatwal"
pull_secret_file = "/root/powervs-clusters/ocp-install-dir-osaka-413-pub-domain-fips/automation/data/pull-secret.txt"
private_key_file = "/root/.ssh/id_rsa"
public_key_file = "/root/.ssh/id_rsa.pub"
[root@rhv-ppc64le-3 powervs-clusters]#

```
getting below errors...
```
Error: [ERROR] Error while creating Load Balanacer Listener err Listener idle connection timeout '0' is invalid. The idle connection timeout should be greater than 50 seconds and less than 2 hours.
│ {
│     "StatusCode": 400,
│     "Headers": {
│         "Cache-Control": [
│             "max-age=0, no-cache, no-store, must-revalidate"
│         ],
│         "Cf-Cache-Status": [
│             "DYNAMIC"
│         ],
│         "Cf-Ray": [
│             "7ddd60bdc9400a9d-IAD"
│         ],
│         "Content-Length": [
│             "437"
│         ],
│         "Content-Type": [
│             "application/json; charset=utf-8"
│         ],
│         "Date": [
│             "Tue, 27 Jun 2023 11:33:56 GMT"
│         ],
│         "Expires": [
│             "-1"
│         ],
│         "Pragma": [
│             "no-cache"
│         ],
│         "Server": [
│             "cloudflare"
│         ],
│         "Strict-Transport-Security": [
│             "max-age=31536000; includeSubDomains"
│         ],
│         "Transaction-Id": [
│             "bf9c6362ae0e3ef430209e29bf144829"
│         ],
│         "X-Content-Type-Options": [
│             "nosniff"
│         ],
│         "X-Correlation-Id": [
│             "bf9c6362ae0e3ef430209e29bf144829"
│         ],
│         "X-Request-Id": [
│             "bf9c6362ae0e3ef430209e29bf144829"
│         ],
│         "X-Xss-Protection": [
│             "1; mode=block"
│         ]
│     },
│     "Result": {
│         "errors": [
│             {
│                 "code": "listener_invalid_idle_connection_timeout",
│                 "message": "Listener idle connection timeout '0' is invalid. The idle connection timeout should be greater than 50 seconds and less than 2 hours.",
│                 "more_info": "https://cloud.ibm.com/docs/vpc?topic=vpc-rias-error-messageslistener_invalid_idle_connection_timeout",
│                 "target": {
│                     "name": "listeners.idle_connection_timeout",
│                     "type": "field"
│                 }
│             }
│         ],
│         "trace": "bf9c6362ae0e3ef430209e29bf144829"
│     },
│     "RawResult": null
│ }
│
│
│   with module.ibmcloud[0].ibm_is_lb_listener.ingress_https_listener_external,
│   on modules/7_ibmcloud/load_balancer.tf line 222, in resource "ibm_is_lb_listener" "ingress_https_listener_external":
│  222: resource "ibm_is_lb_listener" "ingress_https_listener_external" {
│
```
and  using 
```
# ./openshift-install-powervs -version
Version v1.9.0
```
below error
```
Attempt: 5/5
[retry_terraform] Encountered below errors:         (14%)
│ Error: [ERROR] Error while creating Load Balanacer Listener err Listener idle connection timeout '0' is invalid. The idle connection timeout should be greater than 50 seconds and less than 2 hours.
│ Error: job status failed for job id b3cea718-8af1-4b83-9738-9a2787689bf7 with message: imageImport operation failed::download of file rhcos-powervs-images-jp-osa/rhcos-413.92.202306140611-0-powervs.ppc64le.ova.gz from COS region jp-osa failed, error:404 Not Found
[retry_terraform] ERROR: Terraform command failed after 5 attempts! Please check the log files
```

PTAL @yussufsh

--- a/modules/1_prepare/versions.tf
+++ b/modules/1_prepare/versions.tf
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "~> 1.52.0"
+      version = "~> 1.54.0"
     }
     null = {
       source  = "hashicorp/null"

--- a/modules/4_nodes/versions.tf
+++ b/modules/4_nodes/versions.tf
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "~> 1.52.0"
+      version = "~> 1.54.0"
     }
     ignition = {
       source  = "community-terraform-providers/ignition"

--- a/modules/5_install/versions.tf
+++ b/modules/5_install/versions.tf
@@ -26,7 +26,7 @@ terraform {
     }
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "~> 1.52.0"
+      version = "~> 1.54.0"
     }
 
   }

--- a/modules/7_ibmcloud/versions.tf
+++ b/modules/7_ibmcloud/versions.tf
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "~> 1.52.0"
+      version = "~> 1.54.0"
     }
   }
   required_version = ">= 1.2.0"

--- a/versions.tf
+++ b/versions.tf
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "~> 1.52.0"
+      version = "~> 1.54.0"
     }
     random = {
       source  = "hashicorp/random"

